app/web/src/newhotness/Map.vue

```diff
@@ -620,11 +675,8 @@ const renderEdges = (svg: any, edges: layoutLine[]) => {
     .enter()
     .append("path")
     .attr("class", (d: layoutLine) => {
-      const [targetId, sourceId] = d.id.split("-");
-      const isConnectedToSelected = Array.from(selectedComponents.value).some(
-        (component) => targetId === component.id || sourceId === component.id,
-      );
-      return `edge ${isConnectedToSelected ? "connected" : ""}`;
+      const styling = getEdgeStyling(d.id);
+      return `edge ${styling.isConnectedToSelected ? "connected" : ""}`;
```

Comment:
is this just tidying up the code? or do we think this has an impact on the perf?

Reply:
This is just tidying up as that code was getting messy :/